packages/core/src/controllers/verify.ts

```diff
@@ -33,7 +38,23 @@ export class Verify extends IVerify {
       this.removeIframe();
     }
     this.verifyUrl = verifyUrl;
-    await this.createIframe();
+
+    await this.createIframe().catch((error) => {
```

Comment:
Generally not a huge fan of mixing `async/await` and `then/catch` in functions that are already procedural (i.e. not a functional pipe), but if you have a strong preference here OK by me 👍

Reply:
(The alternative being a `try/catch` statement)